index.html

```diff
@@ -0,0 +1,571 @@
+
+Approximately, the process is:
+  - Any elements named `something__likeThis` are **not exported**, unless&#8230;
+    - &#8230;the feature which provides them ("`something`") has `export: true` on its `@core(feature:)` declaration, *or*
+    - &#8230;they are annotated with `@core__export` (or explicitly, `@core__export(isExport: true)`).
+  - Any elements with `normalNames` are **exported**, unless&#8230;
+    - &#8230;the element is annotated with `@core__export(isExport: false)`
+    - &#8230;the element is a directive or directive definition whose name matches the name of a feature, in which case, it is exported only if the entire feature is exported (having been brought in with `@core(feature:, export: true)`).
+
+A formal description is provided by the [is exported](#is-exported) algorithm.
+
+## Roles
+
+```mermaid
+#::[Actors who may be interested in the core schemas]
+graph TB
+  author("👩🏽‍💻 🤖  &nbsp;Author")-->schema(["☉ Core Schema"])
+  schema-->proc1("🤖 &nbsp;Processor")
+  proc1-->output1(["☉ Core Schema[0]"])
+  output1-->proc2("🤖 &nbsp;Processor")
+  proc2-->output2(["☉ Core Schema[1]"])
+  output2-->etc("...")
+  etc-->final(["☉ Core Schema [final]"])
+  final-->core("🤖 Data Core")
+  schema-->reader("👩🏽‍💻  Reader")
+  output1-->reader
+  output2-->reader
+  final-->reader
+```
+
+- **Authors (either human or machine)** write an initial core schema as specified in this document, including versioned `@core` requests for all directives they use
+- **Machine processors** can process core schemas and output new core schemas. The versioning of directives and associated schema elements provided by the `@core` allows processors to operate on directives they understand and pass through directives they do not.
+- **Human readers** can examine the core schema at various stages of processing. At any stage, they can examine the `@core` directives and follow URLs to the specification, receiving an explanation of the requirements of the specification and what new directives, types, and other schema objects are available within the document.
+- **Data cores** can then pick up the processed core schema and provide some data-layer service with it. Typically this means serving the export schema as a GraphQL endpoint, using metadata defined by machinery to inform how it processes operations it receives. However, data cores may perform other tasks described in the core schema, such as routing to backend services, caching commonly-accessed fields and queries, and so on. The term "data core" is intended to capture this multiplicity of possible activities.
+
+## The Basics
+
+Core schemas:
+  1. **must** be valid GraphQL schema documents,
+  2. **must** contain exactly one `SchemaDefinition`, and
+  3. **must** use the `@core` directive on their schema definition to declare any features they reference by using [`@core(feature:)`](#core) to reference a [well-formed feature URL](#feature-urls).
```

Comment:
If you can do it succinctly, a forward reference to the "renaming core itself" section would be nice, since otherwise this statement is a bit too strong.